README.md

```diff
@@ -1 +1,30 @@
-# 2019-top-cat
\ No newline at end of file
+# 2019-top-cat
+
+
+## API scrape
+### *APIScrape/scrape.php*
+
+* Send API request to get the data below
+    * List of cat breeds
+    * All the img src url of all the cat breeds
+    
+* To run the script you must already have a database by the name `cat-test`
+* Then type in the command line 
+**(You must be inside the APIScrape directory)**
+
+    ```$ php scrape.php ```
```

Comment:
Hadn't spotted this earlier! @mallowmew this looks good, but I think the edits you made to the readme in the APIScrape are better. Let's keep both to save time, but be aware that in a large codebase you'd have to be wary of one being updated without the other.